src/js_sync.hpp

```diff
@@ -823,7 +823,13 @@ void SyncClass<T>::populate_sync_config(ContextType ctx, ObjectType realm_constr
             is_partial = Value::validated_to_boolean(ctx, partial_value);
         }
 
-        config.sync_config = std::make_shared<SyncConfig>(shared_user, std::move(raw_realm_url));
+        bool force_open_realms = false;
+        ValueType force_open_realms_value = Object::get_property(ctx, sync_config_object, "forceOpenRealms");
```

Comment:
This seems like an awfully general configuration option name for such a specific purpose.

Reply:
True, maybe `allowOpenPartialSyncURLs` is better.

Reply:
I have on purpose not updated `CHANGELOG.md` or API doc. @kraenhansen isn't happy about underscores.

Reply:
We can update CHANGELOG.md under #internal. I think it's fine to have some way of making unsupported internal APIs clear - e.g. with _. Do we have others like that?